js-miniapp-sample/src/pages/media.js

```diff
@@ -1,14 +1,18 @@
 // @flow
 import React from 'react';
+import ReactPlayerLoader from '@brightcove/react-player-loader';
+
+const onSuccess = function (success) {
+  console.log(success.ref);
+};
 
 const Media = () => {
   return (
-    <video width="100%" controls>
-      <source
-        src="https://multiplatform-f.akamaihd.net/i/multi/will/bunny/big_buck_bunny_,640x360_400,640x360_700,640x360_1000,950x540_1500,.f4v.csmil/master.m3u8"
-        type="application/x-mpegURL"
-      ></source>
-    </video>
+    <ReactPlayerLoader
+      accountId="1752604059001"
```

Comment:
Where does this come from? Is it just a test video provided by brightclove?

Reply:
Yes, I got this info from the official example of brightcove.
https://player.support.brightcove.com/coding-topics/react-player-loader.html